docs/geodjango.rst

```diff
@@ -259,6 +270,7 @@ widget):
 .. code-block:: python
 
     # forms.py
+    from django.template.defaultfilters import safe
```

Comment:
`safe` is meant to be used as a filter in the template. I think we can use `django.utils.html.mark_safe` here instead.